thrift-commons/src/main/thrift/common.thrift

```diff
@@ -33,15 +33,38 @@ struct TSStatus {
   4: optional TEndPoint redirectNode
 }
 
-struct TRegionReplicaSet {
-  1: required binary regionId
-  2: required list<TEndPoint> endpoint
+enum TConsensusGroupType {
+  PartitionRegion,
+  DataRegion,
```

Comment:
？

Reply:
We delete the GroupType defined by Java. Now we maintain TConsensusGroupType by thrift